recipes/spectralib/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.8.1" %}
+
+package:
+  name: spectralib
+  version: {{ version }}
+
+source:
+  fn: spectralib-{{ version }}.tar.gz
+  url: https://github.com/yixuan/spectra/archive/v{{ version }}.tar.gz
+  sha256: 339ae9221309a128b8d937ca59b77d9b30aeceacb4ef2d2df13f6f7cde7fa3f3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Do you really need compilers? The script looks quite simple.

Reply:
no, I've tried this as the windows build keeps failing

Reply:
@xhochy I'm disabling windows for now, I'll re-enable it in the feedstock